src/pages/impact.js

```diff
@@ -5,18 +5,17 @@ import MenuBar from '../components/MenuBar';
 import Hero from '../components/Hero';
 import LineSeparator from '../components/ui-library/LineSeparator';
 import OurSolution from '../components/impact/OurSolution';
+import SDGBlock from '../components/impact/SDGBlock';
 import OurStats from '../components/impact/OurStats';
 import OurContributions from '../components/impact/OurContributions';
 import Footer from '../components/Footer';
 
-// same hero as Model > Companies
-import heroImage from '../assets/modelPage/companies.hero.jpg';
```

Comment:
Are we still using this hero image elsewhere? If not, we should remove from the repo.

Reply:
yes it still use on companies page